src/components/CustomButton.module.scss

```
@@ -0,0 +1,20 @@
+@import "src/theme/mixin";
+
+// TODO #77 style focused state
+.button {
+  @include noSelect; 
+  background-color: var(--vote-button-background);
+  color: var(--vote-button-color);
+  border-radius: 12px;
+  font: inherit;
+  font-weight: bold;
+  border: none;
+  outline: none;
+  padding: 1rem;
+  cursor: pointer;
+  transition: 0.2s ease-out;
+
```

An indication when the button is focused would be nice. The border should be black for light theme and black for the dark one.
```suggestion
  &:focus-visible {
    border-style: solid;
    border-color: var(--vote-button-focused)
  }
```

---

> An indication when the button is focused would be nice. The border should be black for light theme and black for the dark one.

There is an issue for that (#77) as I didn't want to make design assumptions and focus was purposefully disabled previously

---

Since there's a TODO item and an issue for it, I think it's ok to defer this to a later PR.